fossa/pr-check/action.yml

```
@@ -0,0 +1,81 @@
+name: pr check
+
+description: |
+  Check for new issues introduced by a pull request.
+
+  Note: Currently limited to license-related issues.
```

```suggestion
  Check for new license issues introduced by a pull request.
```
Would make it more concise.

---

Indeed.